h2o-py/h2o/estimators/extended_isolation_forest.py

```diff
@@ -0,0 +1,567 @@
+    """
+    Extended Isolation Forest
+
+    Builds an Extended Isolation Forest model. Extended Isolation Forest algorithm samples the training frame
+    and in each iteration builds a tree that partitions the space of the sample observations until
+    it isolates each observation. Length of the path from root to a leaf node of the resulting tree
+    is used to calculate the anomaly score. Anomalies are easier to isolate and their average
+    tree path is expected to be shorter than paths of regular observations.
+    """
+
+    algo = "extendedisolationforest"
+    param_names = {"model_id", "training_frame", "score_each_iteration", "score_tree_interval", "ignored_columns",
+                   "ignore_const_cols", "ntrees", "max_depth", "min_rows", "max_runtime_secs", "seed",
+                   "build_tree_one_node", "sample_size", "extension_level", "col_sample_rate_change_per_level",
+                   "col_sample_rate_per_tree", "categorical_encoding", "stopping_rounds", "stopping_metric",
+                   "stopping_tolerance", "export_checkpoints_dir"}
+
+    def __init__(self, **kwargs):
+        super(H2OExtendedIsolationForestEstimator, self).__init__()
+        self._parms = {}
+        for pname, pvalue in kwargs.items():
+            if pname == 'model_id':
+                self._id = pvalue
+                self._parms["model_id"] = pvalue
+            elif pname in self.param_names:
+                # Using setattr(...) will invoke type-checking of the arguments
+                setattr(self, pname, pvalue)
+            else:
+                raise H2OValueError("Unknown parameter %s = %r" % (pname, pvalue))
+
+    @property
+    def training_frame(self):
+        """
+        Id of the training data frame.
+
+        Type: ``H2OFrame``.
+
+        :examples:
+
+        >>> cars = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/junit/cars_20mpg.csv")
+        >>> predictors = ["displacement","power","weight","acceleration","year"]
+        >>> cars_if = H2OExtendedIsolationForestEstimator(seed = 1234)
+        >>> cars_if.train(x = predictors,
+        ...               training_frame = cars)
+        >>> cars_if.model_performance()
+        """
+        return self._parms.get("training_frame")
+
+    @training_frame.setter
+    def training_frame(self, training_frame):
+        self._parms["training_frame"] = H2OFrame._validate(training_frame, 'training_frame')
+
+
+    @property
+    def score_each_iteration(self):
+        """
+        Whether to score during each iteration of model training.
+
+        Type: ``bool``  (default: ``False``).
+
+        :examples:
+
+        >>> cars = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/junit/cars_20mpg.csv")
+        >>> predictors = ["displacement","power","weight","acceleration","year"]
+        >>> cars_if = H2OExtendedIsolationForestEstimator(score_each_iteration = True,
```

Comment:
There's a lot of line mismatch? I don't know how to explain it--where the next parameter listed in building the estimator isn't directly beneath the line above it (e.g. how `ntrees` isn't directly below `score_each_iteration`). 

Would you like me to go through and fix this as well?

Reply:
Maybe I dont understand. If the parameters setter and getter are in different order, I think it is ok. :)

Reply:
I understand, I will fix the formating of model attributes

Reply:
Fixex